types/qunit/test/global-test.ts

```diff
@@ -373,7 +342,7 @@ QUnit.test( "a test", function( assert ) {
 // declare var Promise: any;
 QUnit.test( "a Promise-returning test", function( assert ) {
   assert.expect( 0 );
-  var thenable = new Promise(function( resolve: any, reject: any ) {
+  var thenable = new Promise<void>(function( resolve: any, reject: any ) {
     setTimeout(function() {
       resolve( "result" );
```

Comment:
It's weird that it doesn't complain about this. I'd expect it to require:
```suggestion
      resolve();
```
Oh, maybe because the `resolve: any` in the args?

Reply:
Likely.

Reply:
I still fixed it.